bin/dsh

```diff
@@ -973,9 +1027,13 @@ EOF
 	install_sshagent_service
 	if_failed "Drude ssh-agent service setup failed."
 
-	echo-green "Configuring host DNS resolver for .drude domain..."
-	echo -e "\n# .drude domain resolution\nnameserver $DRUDE_IP" | sudo tee -a  /etc/resolvconf/resolv.conf.d/head
-	sudo resolvconf -u
+	if ! is_redhat; then
+	  echo-green "Configuring host DNS resolver for .drude domain..."
+	  echo -e "\n# .drude domain resolution\nnameserver $DRUDE_IP" | sudo tee -a  /etc/resolvconf/resolv.conf.d/head
+	  sudo resolvconf -u
+	else
+		sudo systemctl restart NetworkManager.service
```

Comment:
Looking at the lo:drude interface configuration above, it looks like Drude's DNS will be used on that interface. Is this enough to have `*.drude` resolvable on a RHEL host?